bootstrap.d/x11-base.yml

```diff
@@ -152,3 +149,69 @@ packages:
       - args: ['ninja', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: xwayland
+    labels: [aarch64]
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: ports
+      git: 'https://gitlab.freedesktop.org/xorg/xserver.git'
+      tag: 'xwayland-22.1.3'
+      version: '22.1.3'
+    metadata:
```

Comment:
Wrong location, other ports put metadata above the source declaration. For consistency's sake we should do the same here.